.prettierignore

```diff
@@ -0,0 +1,8 @@
+.cache
+build
+build-module
+build-types
+packages/block-serialization-spec-parser/parser.js
+packages/e2e-tests/plugins
```

Comment:
Do you know why this was necessary? There's JavaScript there that seems like it should be formatted. I'll create a PR to remove this and see what happens.

Reply:
There are some test plugins with JS code that not necessarily are compatible with the linting rules established for Gutenberg plugin. Feel free to fix it.

Reply:
I'm trying in https://github.com/WordPress/gutenberg/pull/62572.